sdk/terraform/Azure.ResourceManager.Terraform/tests/Tests/ExportTerraformTests.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.Resources;
+using Azure.ResourceManager.Terraform.Models;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.Terraform.Tests.Tests
+{
+    [TestFixture]
+    public class ExportTerraformTests : TerraformManagementTestBase
+    {
+        private ResourceGroupResource _resourceGroup;
+
+        public ExportTerraformTests() : base(true)
```

Comment:
we need to modify this to 
```suggestion
        public ExportTerraformTests(bool isAsync) : base(isAsync)
```

Reply:
we do this, test framework will produce another version for all your test cases so that one of them is async and the other is sync.
This also requires we must always use the async version of an operation in our sdk.

Reply:
Fixed in f86ae1e58ee